tools/stm32_pinmap_tool.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python3
+############################################################################
```

Comment:
will you change it to the Apache header ?

Reply:
yes

Reply:
Thnks Done